files/image_config/monit/container_checker

```diff
@@ -70,25 +51,60 @@ def get_expected_running_containers():
                     num_asics = multi_asic.get_num_asics()
                     for asic_id in range(num_asics):
                         expected_running_containers.add(container_name + str(asic_id))
+            elif feature_table[container_name]["state"] == 'always_enabled':
+                always_running_containers.add(container_name)
             else:
                 expected_running_containers.add(container_name)
 
-    return expected_running_containers
+    return expected_running_containers, always_running_containers
 
 
-def get_current_running_containers():
+def get_current_running_from_DB(always_running_containers):
     """
-    @summary: This function will get the current running container list by analyzing the
-              output of command `docker ps`.
-    @return:  A set which contains the current running contianers.
+    @summary: This function will get the current running container list
+              from FEATURE table @ STATE_DB, if this table is available.
+    @return:  a tuple
+              First: Return value indicating if info can be obtained from
+              DB or not.
+              Second: A set which contains the current running containers,
+              if this info is available in DB
```

Comment:
It would be better to add a punctuation mark `.` at the end of the sentence.

Reply:
done